### ISSUE TITLE
[terra-tag] [clinical-lowlight-theme] color-contrast accessibility error

### ISSUE THREAD

**Issue by KunigundaLauly** [cerner/terra-core#3442]
# Bug Report

## Description

Terra.validates.element test for the Terra-tag in clinical-lowlight-theme throws an accessibility error for the colors.

## Steps to Reproduce

1. Run a wdio (v6) Terra.validates.element on a module which has a terra-tag.
2. The test will throw the following:
"failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.79 (foreground color: #007ee0, background color: #1d2427, font size: 10.5pt (14px), font 
"html": "<button type=\"button\" class=\"Tag-module__tag___3CjWh Tag-module__is-interactive___2qzIj Tag-module__clinical-lowlight-theme___33N2W 

## Additional Context / Screenshots

![image](https://user-images.githubusercontent.com/47626274/118667508-5e7bb180-b7f4-11eb-9a50-9c5ee72c1c0c.png)

![image](https://user-images.githubusercontent.com/47626274/118666311-5cfdb980-b7f3-11eb-9447-2ba68bc83176.png)

![image](https://user-images.githubusercontent.com/47626274/118667022-ec0ad180-b7f3-11eb-9652-52e0ba9342ce.png)
![image](https://user-images.githubusercontent.com/47626274/118667088-fb8a1a80-b7f3-11eb-87fb-24329a000f24.png)

## Expected Behavior

The wdio tests should not fail.

## Possible Solution

Please fix the color ratio between:
--terra-tag-interactive-color: #007ee0;
--terra-tag-interactive-background-color: #1d2427;
In here:
https://github.com/cerner/terra-core/blob/main/packages/terra-tag/src/clinical-lowlight-theme/Tag.module.scss#L24

## Environment

* Component Name and Version: terra-tag 2.47.0
* Browser Name and Version: Chrome Version 90.0.4430.212 (Official Build) (64-bit)
* Node/npm Version:  N/A
* Webpack Version: N/A
* Operating System and version (desktop or mobile): Windows desktop

**Comment by neilpfeiffer** [comment]
Thanks @KunigundaLauly - while we use the AXE color-contrast checking for WCAG accessibility standards for our "normal light" themes, however, Terra currently [does not apply the AXE color checking](https://github.com/cerner/terra-toolkit/blob/d33d69af3223003a9c909af29529188c2a561ff8/packages/terra-functional-testing/src/services/wdio-terra-service.js#L81-L85) for the `clinical-lowlight-theme` by design.

Cerner UX has done in-depth research into perceptive color in various lighting situations, how the relationships of particular colors appearing on backlit screens change as the ambient and situational room lighting is lowered, as well as options for different "low" light interface theme palettes (e.g. ambient lowlight situations of a nurse unit with lights turned down, no-light radiology reading rooms being even darker and even more white/light sensitive, etc). For example, colors that have low perceptive differences, 80% gray compared to 85% gray, in normal lighting conditions -- become very noticeable in lowlight situations. 

Based on our research into the WCAG regulations for color-contrast, they were created primarily for normal lighting situations. As such, Cerner UX and Human Factors decided to create a lowlight color palette summative user test, at a variety of OSHA defined lighting situations, and validate our current palette for acceptance and perceptive noticeability as a supplement to the WCAG requirements as a our viable baseline, which uses a similar ratio comparison to WCAG but with tested and validated alterations. Since there isn't a current replacement test set to AXE for lowlight, Terra has disabled testing for color contrast. As such, you are fine to as well for your project for the lowlight-theme.
 

Just for comparison, the color used for terra-tag is according to the current palette and used throughout the Terra `clinical-lowlight-theme`:
- [terra-tag](https://engineering.cerner.com/terra-core/components/terra-tag/tag/tag)
https://github.com/cerner/terra-core/blob/2ef331f8df2e3d1bca4762a1c0da0b527855c003/packages/terra-tag/src/clinical-lowlight-theme/Tag.module.scss#L24

- [terra-hyperlink](https://engineering.cerner.com/terra-core/components/terra-hyperlink/hyperlink/hyperlink)
https://github.com/cerner/terra-core/blob/2ef331f8df2e3d1bca4762a1c0da0b527855c003/packages/terra-hyperlink/src/clinical-lowlight-theme/Hyperlink.module.scss#L3

- https://engineering.cerner.com/terra-core/components/terra-paginator/paginator/paginator
https://github.com/cerner/terra-core/blob/2ef331f8df2e3d1bca4762a1c0da0b527855c003/packages/terra-paginator/src/clinical-lowlight-theme/Paginator.module.scss#L8


\
\
Unless there are additional questions or concerns, we will close this issue as won't fix, theme is performing as designed.

**Comment by KunigundaLauly** [comment]
Thank you for the quick response and precise information. I will forward it to my team and I am closing this ticket.
Thank for the help!
